src/polygon/src/polygon.cpp

```diff
@@ -143,7 +144,7 @@ void CheckTopology(const std::vector<glm::ivec3> &triangles,
 
 void CheckGeometry(const std::vector<glm::ivec3> &triangles,
                    const PolygonsIdx &polys, float precision) {
-  std::map<int, glm::vec2> vertPos;
+  std::unordered_map<int, glm::vec2> vertPos;
```

Comment:
ok, this seemingly innocent change is causing bug...

Reply:
Interesting, thanks! I have a plan on the triangulator, but I need to find a couple days to put it into action.